content/blog/colour-scales.md

```diff
@@ -0,0 +1,72 @@
+ - Reverse scale: boolean indicating whether to reverse the colour scale .i.e. map each 0-1 value to the scale's value for 1-value.
+ - Min and max: the expected min and max values for the indicator. These are mapped to the 0 and 1 values of the colour scale, and data points between are mapped to values between 0 and 1 by interpolation. 
+
+<img src="/img/cscale_choro.png" alt="Choropleth" style="width:500px;"/>
+*A choropleth in Naomi showing district-level [ART](https://www.who.int/hiv/topics/treatment/art/en/) coverage using the interpolateViridis colour scale*
+
+However we found that, while this approach was a powerful way for researchers to define how data should be visualised, it was not sufficiently flexible to give users the best results for all datasets.
+ 
+The min-max ranges defined by metadata may not be wide enough to capture all values in a particular dataset, in which case all values which exceed that range will be assigned the same colour, and hence be indistinguishable. Conversely, if a default range is too wide, and all dataset values are clustered within one area of the scale, these may also be difficult to distinguish visually. 
+
+A relevant factor here is that our choropleth data is always a filtered subset of a full dataset, showing values for a particular age range, year, etc, where the user can change which filter set to display using drop-down controls. The metadata range provides a colour scale which is consistent - the same colour corresponds to the same value for each set of filter values. However we found that some filter sets would typically exhibit much narrower indicator ranges than the whole dataset, so that the map rendered could be of little use as a choropleth. 
+
+To solve these issues, we implemented a set of options allowing the user to control the colour range dynamically:
+
+<img src="/img/cscale_options.png" alt="Colour scale user options" style="width:500px;"/>
+
+These are:
+
+ - Static
+    - Default: use range from metadata
+    - Custom: user selects arbitrary values for range
+ - Dynamic
+    - Entire dataset: Use min and max values in actual (unfiltered) dataset
+    - Filtered dataset: Use min and max value in actual dataset, filtered by the current filter set.
+
+Here's an example of how 'Entire dataset' differs from 'Filtered dataset'. In these two images, the colour scale fits the entire dataset, hence is consistent across all filter sets. This means there is typically less colour variation for a given set of filters, but since the colours are fixed to the same values across all sets of filters, it is easy to visually compare the values across both filter sets. 
+<img src="/img/cscale_entire_both.png" alt="Entire dataset colour scale"/>
+<img src="/img/cscale_entire_female.png" alt="Entire dataet colour scale"/>
+
```

Comment:
This might work better if you zoom these down so you can view them side by side without scrolling. Perhaps crop out the side-bar and label/describe the female/both difference another way.

Perhaps the same for the pairs of images that follow...